packages/zapp/console/src/components/flytegraph/ReactFlow/transformDAGToReactFlowV2.tsx

```diff
@@ -78,8 +80,16 @@ const buildReactFlowDataProps = (props: BuildDataProps) => {
   const nodeExecutionStatus = mapNodeExecutionStatus();
 
   // nodeExecutionsById null check is required as on first render it can be undefined
+  const mapNodeExternalResources = () => {
```

Comment:
? What type is it expected to return ? 
currently it is any, and not obvious what will happen if !nodeExecutionsById as there is no specific return in this case.

Reply:
added type and got rid of the function -  retrieving the value inline